lib/govuk_publishing_components/presenters/absolute_links_helper.rb

```diff
@@ -0,0 +1,18 @@
+module GovukPublishingComponents
+  module Presenters
+    class AbsoluteLinksHelper
+      def initialize(environment = GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment)
+        @environment = environment
+      end
+
+      def make_url_absolute_on_production(href)
```

Comment:
We can remove the check for production here by using `Plek.new.website_root` to return the explicit domain of the environment. Do this instead of hardcoding and this fix is actually quite solid. It may need to be extended to cover the report-a-problem form too.